migrations/1_add_user_tables.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS "users"
+(
+  id bigserial NOT NULL,
+  email varchar NOT NULL UNIQUE,
+  username varchar NOT NULL UNIQUE,
```

Comment:
You should consider adding indices on the `email` and `username` columns, since you query on those above.